imports/ui/Dashboard.jsx

```diff
@@ -146,28 +185,38 @@ export const Dashboard = ({
     },
   ];
 
+
   return (
-    <div>
-      <DashboardTable
-        title="Credit"
-        subtitle="every operation that adds account balance"
-        columns={columns(credit)}
-        datasource={credit}
-        balance={creditBalance.toFormat()}
-        newEntryKey="credit"
-        setPayed={setReceived}
-        onClickPayed={markAsPayed}
-      />
-      <DashboardTable
-        title="Debit"
-        subtitle="every operation that subtracts account balance"
-        columns={columns(debit)}
-        datasource={debit}
-        balance={debitBalance.toFormat()}
-        newEntryKey="debit"
-        setPayed={setPayed}
-        onClickPayed={markAsPayed}
-      />
-    </div>
+      <>
+        {!subscriptionHandle.ready() ?
```

Comment:
the ready() has to be called inside the useTracker, because it is the source of reactivity of the publication. If you call it outside the useTracker, the function that fetches the data is never going to be called again when the publication is ready.